microsoft-edge/webview2/roadmap.md

```diff
@@ -26,36 +26,57 @@ The WebView2 team is planning the following major efforts for future updates:
 <!-- ====================================================================== -->
 ## WebView2 Runtime and Installer
 
-Evergreen distribution mode allows you to target or chain-install the WebView2 Runtime onto your users' machines.  The Evergreen WebView2 Runtime and installer has reached General Availability (GA).  For more information, see [Distribute your app and the WebView2 Runtime](concepts/distribution.md).
+Evergreen distribution mode allows you to target or chain-install the WebView2 Runtime onto your users' machines.  The Evergreen WebView2 Runtime and installer has reached General Availability (GA).  For more information, see [Distribute your app and the WebView2 Runtime](./concepts/distribution.md).
 
 
 <!-- ====================================================================== -->
 ## Fixed version
 
-Fixed version distribution mode allows you to package the Microsoft Edge binaries <!--(a specific version of the WebView2 Runtime)--> inside your native application.  The Fixed Version has reached General Availability (GA).  For more information, see [Distribute your app and the WebView2 Runtime](concepts/distribution.md).
+Fixed version distribution mode allows you to package the Microsoft Edge binaries <!--(a specific version of the WebView2 Runtime)--> inside your native application.  The Fixed Version has reached General Availability (GA).  For more information, see [Distribute your app and the WebView2 Runtime](./concepts/distribution.md).
 
 
 <!-- ====================================================================== -->
 ## General Availability
 
 The following technologies have reached General Availability (GA).
 
-### Win32 C/C++
+
+<!-- ------------------------------ -->
+#### Win32 C/C++
 
 The Win32 C/C++ SDK has reached GA.
 
-### .NET
+
+<!-- ------------------------------ -->
+#### .NET
 
 The .NET SDK has reached GA.
 
-### Windows UI Library 2
 
-You can access WebView2 controls in your UWP applications using [Windows UI Library 2 (WinUI 2)](get-started/winui2.md). This has reached GA.
+<!-- ------------------------------ -->
+#### Windows UI Library 2
+
+You can access WebView2 controls in your UWP applications using [Windows UI Library 2 (WinUI 2)](./get-started/winui2.md). This has reached GA.
 
-### Windows UI Library 3
+
+<!-- ------------------------------ -->
+#### Windows UI Library 3
 
 You can access WebView2 controls in your applications using [Windows UI Library 3 (WinUI 3)](/uwp/toolkits/winui3/index) with the Windows App SDK.  This has reached GA.
 
-#### XAML limitation
+###### XAML limitation
 
+This version of WebView 2 will only work on PC-class and HoloLens 2 devices, offering the full range of capability found in the WinUI 3 variant.
```

Comment:
@mikehoffms I think we decided to remove this like "This version of WebView2 ... WinUI3 variant"

Reply:
deleted sent. locally

Reply:
done, see fresh https://github.com/MicrosoftDocs/edge-developer/pull/2567/files
you can close this Outdated comment; open fresh comment if needed